packages/gatsby-source-wordpress/src/steps/create-schema-customization/transform-fields/default-resolver.js

```diff
@@ -45,6 +46,12 @@ export const buildDefaultResolver = transformerApi => (source, _, context) => {
     finalFieldValue = aliasedField2
   }
 
+  if (finalFieldValue?.__typename) {
+    // in Gatsby V3 this property is used to determine the type of an interface field
```

Comment:
@vladar I could minimally reproduce this but I noticed this issue. I couldn't determine what combination of factors was causing the ggl server to resolve the type of interfaces and unions using the `__typename` instead of the `resolveType` fn. It worked as expected in my minimal repro so I just fixed it this way in the WP plugin - not sure if this will affect anyone else but thought you should know

Reply:
I was seeing errors about this in a few places - the message is: `Abstract type \"WpTermNode\" was resolve to a type \"Category\" that does not exist inside schema.`. The `resolveType` fn used to make that into `WpCategory` but schema customization is ignoring it now in favour of the resolved `__typename` field.

Reply:
This sounds really odd. The only `__typename`-related change is that we don't add an implicit `__typename` in Gatsby query anymore. So my best bet is that it was relying on `__typename` in V2 as well but now that `__typename` is not implicitly added it resolves to something else.

We didn't touch logic related to resolving interfaces. I think this deserves some investigation so if you can come up with some repro, please let me know.